cookbook/case_studies/ml_training/mnist_classifier/pytorch_single_node_multi_gpu.py

```diff
@@ -0,0 +1,405 @@
+"""
+Single Node, Multi GPU Training
+--------------------------------
+
+When you need to scale up model training in pytorch, you can use the :py:class:`pytorch:torch.nn.DataParallel` for
+single node, multi-gpu/cpu training or :py:class`pytorch:torch.nn.parallel.DistributedDataParallel` for multi-node,
+multi-gpu training.
+
+This tutorial will cover how to write a simple training script on the MNIST dataset that uses
+:py:class`pytorch:torch.nn.parallel.DistributedDataParallel`, since this is the `recommended way <https://pytorch.org/docs/stable/notes/cuda.html#cuda-nn-ddp-instead>`__
```

Comment:
do you mean this to be distributed or just dataparallel?
Or maybe you should clarify that - distributedDataparallel is a superset of dataparallel?